spec/support/helpers.rb

```diff
@@ -92,14 +92,14 @@ def rm_global_bundler_d_dir
       FileUtils.rm_rf(Helpers.global_bundler_d_dir)
     end
 
-    def with_path_based_gem(source_repo)
+    def with_path_based_gem(source_repo, dir_name = "the_gem")
```

Comment:
I noticed a bunch of the callers were changed (e.g. spec/bundler_inject_spec.rb:121) to set the dir_name at the caller side.  If they are all hardcoded at the caller, do we still need the default value here anymore (hard to tell if they were all changed)?

Reply:
oops. I stated this below.
I think this is only called from a few spots. I will remove the default.